lib/src/test/tokenizer.test.ts

```diff
@@ -27,26 +27,26 @@ const languageFiles = {
 } as {[key: string]: string};
 
 const tokenLength = {
-  "../samples/bash/caesar.sh": 1185,
+  "../samples/bash/caesar.sh": 1179,
   "../samples/c/caesar.c": 582,
-  "../samples/c-sharp/Caesar.cs": 606,
+  "../samples/c-sharp/Caesar.cs": 603,
   "../samples/char/caesar.txt": 3700,
-  "../samples/cpp/caesar.cpp": 801,
+  "../samples/cpp/caesar.cpp": 795,
   "../samples/elm/Caesar.elm": 753,
   "../samples/go/Caesar.go": 1032,
   "../samples/groovy/caesar.groovy": 282,
-  "../samples/java/Caesar.java": 522,
+  "../samples/java/Caesar.java": 519,
   "../samples/javascript/sample.js": 861,
-  "../samples/python/caesar.py": 309,
+  "../samples/python/caesar.py": 306,
   "../samples/php/caesar.php": 411,
-  "../samples/modelica/sample.mo": 7542,
-  "../samples/r/caesar.R": 594,
+  "../samples/modelica/sample.mo": 7533,
+  "../samples/r/caesar.R": 585,
   "../samples/rust/caesar.rs": 774,
   "../samples/scala/Caesar.scala": 366,
-  "../samples/sql/sample.sql": 543,
-  "../samples/tsx/sample.tsx": 1659,
-  "../samples/typescript/caesar.ts": 378,
-  "../samples/verilog/module.v": 2484
+  "../samples/sql/sample.sql": 540,
+  "../samples/tsx/sample.tsx": 1656,
+  "../samples/typescript/caesar.ts": 375,
+  "../samples/verilog/module.v": 2448
 } as {[key: string]: number};
```

Comment:
What was the purpose of this particular test (checking the token length) in addition to the tokenizer snapshots?

You've added this in a previous PR, but I see this will change often when the tokenizers change and might me extra manual work - whereas the snapshots can be automatically updated if they change.

Reply:
I made this test to check if my changes to the tokenizer did change the output.  but at that time, I didn't know what the snapshots did. So this test is indeed redundant and I removed it.